libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -45,14 +45,31 @@ func setupIPv6BridgeNetFiltering(config *networkConfiguration, _ *bridgeInterfac
 	return nil
 }
 
+func LoadBridgeModules() error {
+	moduleMap := map[string]string {
+		"arp_probe":                "bridge",
+		"bridge-nf-call-iptables":  "br_netfilter",
+	}
+
+	for fPath, modName := range moduleMap {
+		fullPath := "/proc/sys/net/bridge/" + fPath
+		if _, err := os.Stat(fullPath); err != nil {
+			if out, err := exec.Command("modprobe", "-va", modName).CombinedOutput(); err != nil {
+				log.G(context.TODO()).WithError(err).Errorf("Running modprobe %s failed with message: %s", modName, out)
+				return fmt.Errorf("cannot restrict inter-container communication: modprobe %s failed: %w", modName, err)
+			}
+		}
+	}
+	return nil
+}
+
 // Enable bridge net filtering if not already enabled. See GitHub issue #11404
 func enableBridgeNetFiltering(nfParam string) error {
-	if _, err := os.Stat("/proc/sys/net/bridge"); err != nil {
-		if out, err := exec.Command("modprobe", "-va", "bridge", "br_netfilter").CombinedOutput(); err != nil {
-			log.G(context.TODO()).WithError(err).Errorf("Running modprobe bridge br_netfilter failed with message: %s", out)
-			return fmt.Errorf("cannot restrict inter-container communication: modprobe br_netfilter failed: %w", err)
-		}
+	err := LoadBridgeModules()
+	if err != nil {
```

Comment:
To limit the scope of this `err`, and we try to be fussy about such things in new/changed code ... 

```suggestion
	if err := LoadBridgeModules(); err != nil {
```